cms/documents/migrations/0002_alter_customdocument_file_size.py

```diff
@@ -1,17 +0,0 @@
-# Generated by Django 5.1.2 on 2024-11-01 17:01
-
-from django.db import migrations, models
-
-
-class Migration(migrations.Migration):
-    dependencies = [
-        ("documents", "0001_initial"),
-    ]
-
-    operations = [
-        migrations.AlterField(
-            model_name="customdocument",
-            name="file_size",
-            field=models.PositiveBigIntegerField(editable=False, null=True),
-        ),
-    ]
```

Comment:
please do not delete migrations once they are on `main`. They will have been applied in different environments and you end with a dirty history

Reply:
ok 🫡 I did have to manually add it back as the main was merged in case that may be a problem in future.